Documentation/engineering/updating-ryujit.md

```diff
@@ -4,9 +4,9 @@ Following steps are necessary to pick up a new version of RyuJIT code generation
 
 1. From the master branch of the CoreCLR repo, copy header files that are part of the contract with RyuJIT from `src\inc` on the CoreCLR side, to `src\JitInterface\src\ThunkGenerator` on the CoreRT side.
 2. Inspect the diffs
-  1. If an enum was modified, port the change to the managed version of the enum manually.
-  2. If a JitInterface method was added or changed, update `src\JitInterface\src\ThunkGenerator\ThunkInput.txt` and run the generation script next to the file to regenerate `CorInfoBase.cs` and `jitinterface.h`. Update the managed implementation of the method in `CorInfoImpl.cs` manually.
-  3. If the JitInterface GUID was updated, update it in `src\Native\jitinterface\jitwrapper.cpp`
+    1. If an enum was modified, port the change to the managed version of the enum manually.
+    2. If a JitInterface method was added or changed, update `src\JitInterface\src\ThunkGenerator\ThunkInput.txt` and run the generation script next to the file to regenerate `CorInfoBase.cs` and `jitinterface.h`. Update the managed implementation of the method in `CorInfoImpl.cs` manually.
+    3. If the JitInterface GUID was updated, update it in `src\Native\jitinterface\jitwrapper.cpp`
 3. Go to https://dotnet.myget.org/gallery/dotnet-core and find the latest version of the Microsoft.NETCore.Jit package. Update the version number in dependencies.props at the root of the repo.
```

Comment:
The URL can be a direct https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.NETCore.Jit link so save you a few clicks